doc/classes/RigidBody2D.xml

```diff
@@ -378,74 +381,74 @@
 		<member name="contacts_reported" type="int" setter="set_max_contacts_reported" getter="get_max_contacts_reported">
-			The maximum contacts to report. Bodies can keep a log of the contacts with other bodies, this is enabled by setting the maximum amount of contacts reported to a number greater than 0.
+			The maximum number of contacts to report. Default value: [code]0[/code].
 		</member>
 		<member name="continuous_cd" type="int" setter="set_continuous_collision_detection_mode" getter="get_continuous_collision_detection_mode" enum="RigidBody2D.CCDMode">
-			If [code]true[/code] continuous collision detection is used. Default value: [code]false[/code]
-			Continuous collision detection tries to predict where a moving body will collide, instead of moving it and correcting its movement if it collided. Continuous collision detection is more precise, and misses less impacts by small, fast-moving objects. Not using continuous collision detection is faster to compute, but can miss small, fast-moving objects.
+			Continuous collision detection mode. Default value: [code]CCD_MODE_DISABLED[/code].
+			Continuous collision detection tries to predict where a moving body will collide instead of moving it and correcting its movement after collision. Continuous collision detection is slower, but more precise and misses fewer collisions with small, fast-moving objects. Raycasting and shapecasting methods are available. See [code]CCD_MODE_[/code] constants for details.
 		</member>
 		<member name="custom_integrator" type="bool" setter="set_use_custom_integrator" getter="is_using_custom_integrator">
-			If [code]true[/code] internal force integration will be disabled (like gravity or air friction) for this body. Other than collision response, the body will only move as determined by the [method _integrate_forces] function, if defined.
+			If [code]true[/code] internal force integration is disabled for this body. Aside from collision response, the body will only move as determined by the [method _integrate_forces] function.
 		</member>
 		<member name="friction" type="float" setter="set_friction" getter="get_friction">
-			The body friction, from 0 (frictionless) to 1 (max friction).
+			The body's friction. Values range from [code]0[/code] (frictionless) to [code]1[/code] (maximum friction). Default value: [code]1[/code].
 		</member>
 		<member name="gravity_scale" type="float" setter="set_gravity_scale" getter="get_gravity_scale">
-			This is multiplied by the global 2D gravity setting found in "Project &gt; Project Settings &gt; Physics &gt; 2d" to produce RigidBody2D's gravity. E.g. a value of 1 will be normal gravity, 2 will apply double gravity, and 0.5 will apply half gravity to this object.
+			Multiplies the gravity applied to the body. The body's gravity is calculated from the "Default Gravity" value in "Project &gt; Project Settings &gt; Physics &gt; 2d" and/or any additional gravity vector applied by [Area2D]\s. Default value: [code]1[/code].
```

Comment:
`[Class]\ s`

Reply:
What's with this weird construct though? It should be `[Class]s` and both DocData and makerst should be fixed to handle it properly if they don't.

Reply:
[Class] converts to `:ref:'Class <Class>'`

From: http://www.sphinx-doc.org/en/stable/rest.html
>it must be separated from surrounding text by non-word characters. Use a backslash escaped space to work around that: thisis\ \*one\*\ word.

Last time I check:
`:ref:'Class <Class>' s` became `Class s`.
`:ref:'Class <Class>'s` was broken.
`:ref:'Class <Class>'\ s` became `Classs`.

Reply:
Without escaping:
![image](https://user-images.githubusercontent.com/26753932/31849996-a4cad97e-b675-11e7-9d1c-f705d935d4d5.png)

With escaping:
![image](https://user-images.githubusercontent.com/26753932/31850000-b27f1a6c-b675-11e7-8ca5-eb20413b2563.png)

Reply:
Yeah, but that's why we should handle that in `makerst.py` and not in the XML itself. It's easy enough to detect `[.*][a-zA-Z0-9]` in a string and output the necessary escape accordingly.